packages/backend/package.json

```diff
@@ -50,7 +50,7 @@
     "@graphql-tools/load": "^7.1.9",
     "@graphql-tools/schema": "^8.1.2",
     "@graphql-tools/utils": "^8.5.3",
-    "@interledger/pay": "^0.3.3",
+    "@interledger/pay": "^0.4.0-alpha.0",
```

Comment:
Even though we're helped by yarn zero install, I think we should use the exact version for a prerelease (to avoid breaking changes in `0.4.0-alpha.1`):
```
yarn workspace backend add @interledger/pay@0.4.0-alpha.0
```
https://docs.npmjs.com/cli/v6/using-npm/semver#prerelease-tags